app/scripts/migrations/047.js

```diff
@@ -0,0 +1,40 @@
+const version = 47
+import { cloneDeep } from 'lodash'
+
+/**
+ * Stringify numbers:
+ *  - PreferencesController.frequentRpcListDetail item chainId
+ *  - TransactionsController.transactions item metamaskNetworkId
+ */
+export default {
+  version,
+  migrate: async function (originalVersionedData) {
+    const versionedData = cloneDeep(originalVersionedData)
+    versionedData.meta.version = version
+    const state = versionedData.data
+    versionedData.data = transformState(state)
+    return versionedData
+  },
+}
+
+function transformState (state) {
+
+  const frequentRpcListDetail = state?.PreferencesController?.frequentRpcListDetail
```

Comment:
I don't think this migration is required. The `chainId` is already a string in all cases